src/streams.h

```diff
@@ -629,6 +629,20 @@ class CBufferedFile
         return true;
     }
 
+    //! Try to get up to the requested length bytes available in the buffer,
+    //! starting at stream offset m_read_pos, filling the buffer if necessary.
```

Comment:
I think "filling the buffer if necessary." is confusing - we _only_ fill the buffer if `m_read_pos == nSrcPos`, whereas I'd interpret "if necessary" as more broad than that.